src/bindings/python/requirements_test.txt

```diff
@@ -2,7 +2,7 @@
 numpy
 bandit
 black
-flake8
+flake8<=7.0.0
```

Comment:
should we set upper bound in dedicated constraints.txt file?

Reply:
I don't think it's warranted in this case, because the thing that `constraints.txt` files help the most with is when the requirements are declared multiple times in different `requirements.txt` files. In this case `flake8` requirements are located in a single `requirements.txt` file, so I believe adding `constraints.txt` would bring no benefits.